docs/pages/docs/Dart API/manager.md

```diff
@@ -60,6 +60,12 @@ We can now use them in a query like this:
 In this example, had we not specified a custom name for the reference, the code generator would have named both filtersets `userRefs` for both references to the `User` table. This would have caused a conflict. By specifying a custom name, we can avoid this issue.
 
 
+### Name Clashes
```

Comment:
Imo this could be a `####` heading, then it shows neatly under the code snippet using the annotation.

Reply:
Giving this an anchor also gives us a opportunity to reference drift.simonbinder.eu/docs/manager/#name-clashes in the error message.

Reply:
Done :)